lib/jekyll/feed_meta_tag.rb

```diff
@@ -2,7 +2,15 @@ module Jekyll
   class FeedMetaTag < Liquid::Tag
     def render(context)
       @context = context
-      %(<link type="application/atom+xml" rel="alternate" href="#{url}/#{path}" title="#{config["name"]}" />)
+      link_attributes = [
+        'type="application/atom+xml"',
+        'rel="alternate"',
+        %(href="#{url}/#{path}")
+      ]
+      if title
+        link_attributes << %(title="#{title}")
```

Comment:
Not sure if it makes sense here, but [here's what we did for Jekyll avatar](https://github.com/benbalter/jekyll-avatar/blob/master/lib/jekyll-avatar.rb#L14-L33) to solve a similar problem.